root/layout/components/sidebar/ReleaseGroupSidebar.js

```diff
@@ -40,10 +40,10 @@ component ReleaseGroupSidebar(
         </div>
       ) : null}
 
-      {nonEmpty(firstReleaseGid) ? (
+      {hasReleases ? (
         <PlayOnListenBrainzButton
-          entityType="release"
-          mbids={firstReleaseGid}
+          entityType="album"
+          mbids={gid}
```

Comment:
I noticed that in this case the `gid` is URI-encoded, while in (all?) other cases it's not. We should probably perform URI-encoding inside `getListenBrainzLink`. (It doesn't matter if we "double-encode" here, since a real GUID wouldn't contain any encoded characters to begin with.)

Reply:
Seems sensible, done :)